mne/viz/topo.py

```diff
@@ -152,6 +153,11 @@ def _plot_topo(info, times, show_func, click_func=None, layout=None,
     """Helper function to plot on sensor layout."""
     import matplotlib.pyplot as plt
 
+    if layout.kind == 'custom':
+        layout = copy(layout)
+        for i_dim in layout.pos.T[:2]:
+            i_dim -= i_dim.min(0)
+            i_dim /= (i_dim.max(0) - i_dim.min(0))
```

Comment:
Like this (except the background appears to be white).
![figure_1](https://cloud.githubusercontent.com/assets/3456414/23510224/f2d7d862-ff58-11e6-9fd6-6f4d1c84e135.png)